dts/bindings/bluetooth/silabs,bt-hci.yaml

```diff
@@ -0,0 +1,13 @@
+description: Bluetooth HCI on Silabs boards
+
+compatible: "silabs,bt-hci"
+
+include: bt-hci.yaml
+
+properties:
+  bt-hci-name:
+    default: "sl:bt"
+  bt-hci-bus:
+    default: "BT_HCI_BUS_UART"
```

Comment:
Just to be curious, this driver does not use UART. What is the meaning of `BT_HCI_BUS_UART`? (this value is not a new thing, it was already there)

Reply:
> Just to be curious, this driver does not use UART. What is the meaning of `BT_HCI_BUS_UART`? (this value is not a new thing, it was already there)

I asked the same question here: https://github.com/zephyrproject-rtos/zephyr/pull/72323#issuecomment-2143776113

It looks like it was wrong from the start. It should likely be IPM (if the controller is on another core) or VIRTUAL (if the controller is on the same core).

Reply:
So, it should be `VIRTUAL`.

Reply:
Regarding where these come from, they're purely informational and map to the BTSNoop protocol that's supported by the Linux (BlueZ) btmon tool:
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/include/net/bluetooth/hci.h#n58
They map to the "Bus" parameter of the "New Index" BTSnoop packet:
https://git.kernel.org/pub/scm/bluetooth/bluez.git/tree/doc/btsnoop.txt

More info on how to use this with Zephyr: https://docs.zephyrproject.org/latest/connectivity/bluetooth/bluetooth-tools.html#embedded-hci-tracing